api/controllers/updateProfile.py

```diff
@@ -0,0 +1,56 @@
+import os
+
+from flask import Blueprint, request, jsonify
+from api.utils.errors import ErrorResponse
+from flask import current_app as app
+from api.helpers.uploads import saveToImage
+from api.utils.firebaseUploader import fileUploader
+from api.schemas.errors import (
+    PayloadNotFound,
+    ImageNotFound,
+    UserNotFound,
+    ExtensionNotFound,
+    PNGNotFound
+)
+from api.schemas.user import UpdateUserSchema
+from api.models.user import User
+
+router = Blueprint('updateUserProfile', __name__)
+
+
+@router.route('/profileImage')
+def update_profile_image():
+    try:
+        data = request.get_json()['data']['attributes']
+    except Exception:
+        return ErrorResponse(PayloadNotFound().message, 422, {'Content-Type': 'application/json'}).respond()
+
+    if not data['image']:
+        return ErrorResponse(ImageNotFound().message, 422, {'Content-Type': 'application/json'}).respond()
+
+    if not data['extension']:
+        return ErrorResponse(ExtensionNotFound().message, 422, {'Content-Type': 'application/json'}).respond()
+
+    if data['extension'] != 'png':
```

Comment:
just asking, why are going for only png and no jpg?

Reply:
As the libraries we are using saving the background in the format of `png` so to make the system consistent i have written this line. 
Also founded this online:
The biggest advantage of PNG over JPEG is that the compression is lossless, meaning there is no loss in quality each time it is opened and saved again. PNG also handles detailed, high-contrast images well.